.travis.yml

```diff
@@ -26,9 +26,6 @@ install:
 script:
   - npm run-script coverage
 
-after_script:
-  - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
```

Comment:
These instructions are for private repositories. For public ones you don't need a token. Also note that you're owner of this repo, so you should be able to enable it in coveralls as well.

I would focus on that error you're getting, I have no idea why that happens. I can take a look if you cannot figure it out.